public/home.html

```diff
@@ -168,7 +168,7 @@ <h2>What is Sustainable Development Goal 11?</h2>
 										SDSN Canada? Check out
 										the website here:</p>
 									<a href="https://uwaterloo.ca/sustainable-development-solutions-network-canada/">
-										<img src="public/images/sdsn-canada-logo.jpg" style="width: 8rem;">
+										<img src="public/images/sdsn-canada-logo.jpg" style="height: 8rem;">
```

Comment:
Great call using height as the common denominator to resolve that oddly shaped image. Take a look at the issue specs, we'll want to replace the .jpg with the .png from the email I sent you. You'll notice a difference as we zoom the image like below. I've emailed the png for reference

![image](https://user-images.githubusercontent.com/74987075/125863295-5d978675-2cf2-40ae-90c8-4be810b31ddd.png)

Reply:
Thanks for the email, I will integrate it soon.

Reply:
resolved! awesome work